src/modules/fancyzones/lib/ZoneSet.cpp

```diff
@@ -597,7 +597,7 @@ bool ZoneSet::CalculateCustomLayout(Rect workArea, int spacing) noexcept
     wil::unique_cotaskmem_string guuidStr;
     if (SUCCEEDED_LOG(StringFromCLSID(m_config.Id, &guuidStr)))
     {
-        const std::wstring guuid = guuidStr.get();
+        const std::wstring guuid = guuidStr.get(); 
```

Comment:
Extra space, plus I think we should use `SUCCEEDED` macro instead of `SUCCEEDED_LOG`.

Reply:
Done